CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for crate
 Unreleased
 ==========
 
+ - Testing: Expose ``verbose`` keyword argument on testing layer which allows
```

Comment:
suggestion:

Testing: Made Crate test layer logging less verbose (hide Crate startup logs) and added ``verbose`` keyword argument to layer to control its verbosity.